src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final String[] colors = {"red","blue", "yellow", "green", "orange", "purple"};
```

Comment:
create Enum `Color` instead of massive of String

Reply:
@Ivan95kos thank you friend for your help, maybe today or tomorrow I try to do it with all your indications.